examples/mpibench/test.bats

```diff
@@ -45,9 +45,10 @@ check_process_ct () {
 
 # one from "Single Transfer Benchmarks"
 @test "${ch_tag}/pingpong (guest launch)" {
+    openmpi_or_skip
```

Comment:
Why skip for MPICH?

Reply:
Now that we build MPICH to use SLURM as the process manager it no longer compiles `mpiexec` to launch within the guest.

From their README:
```
slurm
-----
Slurm is an external process manager not distributed with
MPICH. MPICH's default process manager, hydra, has native support
for Slurm and you can directly use it in Slurm environments (it will
automatically detect Slurm and use Slurm capabilities). However, if
you want to use the Slurm-provided "srun" process manager, you can use
the "--with-pmi=slurm --with-pm=no" option with configure. Note that
the "srun" process manager that comes with Slurm uses an older PMI
standard which does not have some of the performance enhancements that
hydra provides in Slurm environments.
```

Reply:
Note that prior to this PR `mpirun` was hardcoded in these examples. So it didn't work with MPICH anyway.

Reply:
Aha lol. Maybe comment that?